docs/artifacts.md

```diff
@@ -1,3 +1,6 @@
+Authors: Ayaz Akram
```

Comment:
Does this syntax render correctly?

I would have expected that you had to use the yaml front matter syntax:

```
---
Authors:
  - Ayaz Akram
  - Jason Lowe-Power
---
```

I could be wrong, though :)

Reply:
The markdown syntax seemed to render correctly to me. But, I liked the yaml syntax more. So, I have pushed a new commit with updated syntax.